backend/api/migrations/20220120034428_set_is_complete_on_modules.sql

```diff
@@ -0,0 +1,8 @@
+update jig_data_module
```

Comment:
`is_complete` wasn't being cloned when a JIG was published from a draft. This migration is needed to retroactively set the is_complete field for published modules which have content set.